docs/data/joy/customization/introduction/introduction.md

```diff
@@ -0,0 +1,118 @@
+# Introduction
+
+<p class="description">Learn the ways to customize Joy UI.</p>
+
+<!-- The purpose of this page is to give the overall customization alternatives to developers without providing too much details -->
+<!-- Some examples and demos are provided to give the sense of what it looks like and then lead each part to another page for more technical details and more examples -->
+
+The customization can be categorized in 2 buckets:
+
+- Global customization (theming)
+- One-off customization (per instance)
+
+## Theming
+
+If you want to apply the style to every instance of the components, theming is the right choice. The benefit of theming is that your implementation does not change when you switch between themes.
+
+Here are some examples that replicate the popular designs (only the light mode):
+
+{{"demo": "ButtonThemes.js", "hideToolbar": true}}
+
+Theming can be categorized into 2 parts, **tokens** and **components**.
+
+### Tokens theming
+
+The tokens (design tokens) are meaningful variables that refer to the generic usage of the application. The idea of the tokens is to create consistency for the application.
+
+:::warning
+Customizing the theme tokens affects all of the components that consume them. If you want to apply a style to a specific component, take a look at [**Components theming**](#components-theming) instead.
+:::
+
+Joy has 8 independent categories:
+
+- palette (related to colors)
+- fontSize
+- fontFamily
+- fontWeight
+- lineHeight
+- letterSpacing
+- radius
+- shadow
+
+To **override the default tokens**, always use the `extendTheme` function. The specified tokens will be deeply merged to the default theme.
+
+```js
+import { CssVarsProvider, extendTheme } from '@mui/joy/styles';
+
+const theme = extendTheme({
+  colorSchemes: {
+    light: {
+      palette: {
+        // affects all Joy components that has `color="primary"` prop.
+        primary: {
+          50: '#fffbeb',
+          100: '#fef3c7',
+          200: '#fde68a',
+          // 300, 400, ..., 800,
+          900: '#78350f',
+        },
+      },
+    },
+  },
+  fontFamily: {
+    display: 'Inter var, var(--joy-fontFamily-fallback)',
```

Comment:
```suggestion
    display: 'Inter, var(--joy-fontFamily-fallback)',
```

Reply:
It is not a typo. I got `Inter var` from tailwind CSS website.